recipes/defiant/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${C_INCLUDE_PATH}:${PREFIX}/include
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+mkdir -p ${PREFIX}/bin
+
+unzip defiant.zip
+
+# Fix defiant.c for OSX
+if [ `uname` == Darwin ]; then
+    sed -i.bak 's/restrict argv\[\]/argv\[\]/g' defiant.c
+fi
+
+# Compile binaries
+${CC} -O4 -o defiant defiant.c -Wall -pedantic -std=gnu11 -lm -fopenmp -I${PREFIX}/include
+mv regions_of_interest regions_of_interest.c
+${CC} -o roi regions_of_interest.c -lm -Wall -std=gnu11 -Wextra -pedantic -Wconversion
+
```

Comment:
Use CFLAGS and LDFLAGS as well if it works. They some optimization flags that can be useful.